.changeset/chatty-flowers-poke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-frontend/application-shell': minor
+'@commercetools-frontend/application-shell-connectors': minor
+---
+
+Logged-in [user data](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#user) that can be accessed from the [application context](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#useapplicationcontext) can now optionally include new info for SSO logged-in users (Identity Provider OIDC token claims).
```

Comment:
Not sure if this is enough.
I would expect for us to have a follow-up task to update the application context's user [docs](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-shell-connectors#user). Maybe also a release note?

Reply:
Yes as a follow up we can document these new fields.